src/components/SiteDetail.js

```diff
@@ -2,26 +2,40 @@ import React, { Component } from 'react';
 
 import summary from '../apis/summary';
 
+import ContactIcon from '@material-ui/icons/Email';
+import { ThemeProvider } from 'styled-components/macro';
 import { withStyles } from '@material-ui/core/styles';
 import Typography from '@material-ui/core/Typography';
-import Grid from '@material-ui/core/Grid';
 import Box from '@material-ui/core/Box';
-import Button from '@material-ui/core/Button';
 import Paper from '@material-ui/core/Paper';
 
 import { TextLoader } from './Loader';
+import { ButtonStyle } from './Button';
 
 import PropTypes from 'prop-types';
 
+const theme = {
```

Comment:
For this, you should create a file at the top level, `src/theme.js` or similar, and export this object. This way, anyone can come back to it by typing in "theme" in the finder autocomplete.